### ISSUE TITLE
autoFocus not working for clone URL textbox when not signed into GitHub.com

### ISSUE THREAD

**Issue by katrinleinweber** [desktop/desktop#8804]
### Describe the bug

See title.

### Version & OS

GHD 2.2.4 on Windows 10 Enterprise (build 17763)

### Steps to reproduce the behavior

1. On a new system, install GHD, but don't sign it into a GitHub.com (or Enterprise Server) account
1. Press `Ctrl`+`Shift`+`O` => Focus is on `GitHub.com` tab
1. Switch to `URL` tab & clone something
1. Press `Ctrl`+`Shift`+`O` again to clone something else
1. Check focus (blue rim)

### Expected behavior

Focus should be on the URL textbox.

### Actual behavior

Focus is on `URL` tab. The URL textbox is only focused, if the dialog opens in the `GitHub.com` tab and one manually switches to the `URL` tab.

### Screenshots

![grafik](https://user-images.githubusercontent.com/9948149/70980824-3db9a480-20b4-11ea-9127-4900a667f42b.png)

### Logs

Nothing about "focus" in there.

### Additional context

https://github.com/desktop/desktop/blob/bded16772fd3405aee4875930ed6a4432ed3a076/app/src/ui/clone-repository/clone-generic-repository.tsx#L37-L40

does not seem to be triggered when the app isn't signed into any account.

**Comment by outofambit** [comment]
Hi @katrinleinweber and thank you for this great writeup! I've run into this myself so I'm going to mark this as a bug.
